### PR TITLE
fix: clean up README

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,8 @@ Code to deal with kafka
 
 Originally created by the HPC team of Ghent University (http://ugent.be/hpc).
```
Comment (wdpypere) [inline README.md]: I prefer https://www.ugent.be/hpc/

File: README.md [hunk 2]
```diff
@@ -4,15 +4,8 @@ Code to deal with kafka
...
-
-# Documentation
-https://jenkins1.ugent.be/job/vsc-kafka-python36/Documentation/
-
 # License
-All rights reserved.
+GNU Public License v2
 
 # Acknowledgements
 vsc-kafka was created with support of [Ghent University](http://www.ugent.be/en),
```
Comment (wdpypere) [inline README.md]: consistency https://www.ugent.be/hpc/

Comment (itkovian) [reply README.md]: That's not the UGent, but the HPC team.

Comment (wdpypere) [reply README.md]: true, I missed that. but https is good practice.

File: README.md [hunk 3]
```diff
@@ -2,21 +2,14 @@
...
-All rights reserved.
+GNU Public License v2
 
 # Acknowledgements
-vsc-kafka was created with support of [Ghent University](http://www.ugent.be/en),
+vsc-kafka was created with support of [Ghent University](https://www.ugent.be/en),
 the [Flemish Supercomputer Centre (VSC)](https://vscentrum.be/nl/en),
-the [Hercules foundation and the Department of Economy](http://www.herculesstichting.be/in_English),
-and [the Department of Economy, Science and Innovation (EWI)](http://www.ewi-vlaanderen.be/en).
+the [Hercules foundation and the Department of Economy](https://www.herculesstichting.be/in_English),
```
Comment (wdpypere) [inline README.md]: this can't be right. Hercules is long long gone.

File: lib/vsc/kafka/cli.py [hunk 4]
```diff
@@ -175,3 +180,88 @@ def do(self, dry_run):
...
+    def convert_msg(self, msg):
+        """
+        Process msg as JSON.
+        Return None on failure.
+        """
+        value = msg.value
+        if value:
+            try:
+                event = json.loads(value)
+            except ValueError:
```
Comment (wdpypere) [inline lib/vsc/kafka/cli.py:220]: does IO error also give ValueError? Probably does not matter. it will fail hard if it does.

File: test/cli.py [hunk 5]
```diff
@@ -33,3 +40,92 @@ def test_make_time(self):
...
+
+class TestConsumer(TestCase):
+    def setUp(self):
+        """Prepare test case."""
+        super(TestConsumer, self).setUp()
+
+        sys.argv = [
+            'name', '--debug',
+            '--brokers=serv1,serv2',
+            '--ssl=ssl1=sslv1,ssl2=sslv2',
```
Comment (wdpypere) [inline test/cli.py:52]: what do these options mean?  just for testing?  sslv1 and sslv2 should not be used anywhere.

Comment (itkovian) [reply test/cli.py:52]: It's just strings for testing.

**Pull request merged**